onnxruntime/core/providers/cuda/activation/activations.cc

```diff
@@ -64,29 +64,33 @@ namespace cuda {
   UNARY_ACTIVATION_OP_VERSIONED_TYPED(name, startver, endver, double)       \
   UNARY_ACTIVATION_OP_VERSIONED_TYPED(name, startver, endver, BFloat16)
 
-#define UNARY_ACTIVATION_OP_HFD(name, ver)        \
-  UNARY_ACTIVATION_OP_TYPED(name, ver, MLFloat16) \
-  UNARY_ACTIVATION_OP_TYPED(name, ver, float)     \
-  UNARY_ACTIVATION_OP_TYPED(name, ver, double)    \
+#define UNARY_ACTIVATION_OP_HFD_WITH_BF16(name, ver) \
```

Comment:
Could you explain the renaming? Is it necessary? It did contain bfloat16 already before the change right?